### ISSUE TITLE
Support for Svelte v5

### ISSUE THREAD

**Issue by TheComputerM** [himynameisdave/svelte-flex#65]
Does this project plan to support svelte v5?

**Comment by TheComputerM** [comment]
Sorry, support as in use the new runes api?

**Comment by himynameisdave** [comment]
Yes, the plan is to fully support the new Runes API! Feel free to open a PR as I don't know when I'll have time to get around to this.
